packages/react/ds/src/select/select.stories.ts

```diff
@@ -0,0 +1,206 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Select } from './select.js';
+
+const meta = {
+  title: 'Form/Select',
+  parameters: {
+    docs: {
+      description: {
+        component: 'Use this select component where it is appropiate',
+      },
+    },
+  },
+  component: Select,
+} satisfies Meta<typeof Select>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  argTypes: {
+    id: {
+      description: 'A unqiue valued used to identify the component',
+      control: 'text',
+      table: {
+        category: 'ID',
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+      },
+    },
+    options: {
+      description: 'A list of labels and values for each option',
+      control: 'object',
+      table: {
+        category: 'Options',
+        type: { summary: 'Options' },
+      },
+    },
+    error: {
+      description: 'Error message for the select',
+      control: 'object',
+      table: {
+        category: 'Error',
+        type: { summary: 'ErrorText' },
+      },
+    },
+    hint: {
+      description: 'Hint message for the select',
+      control: 'object',
+      table: {
+        category: 'Hint',
+        type: { summary: 'HintText' },
+      },
+    },
+    label: {
+      description: 'Label associated for the select',
+      control: 'object',
+      table: {
+        category: 'Label',
+        type: { summary: 'Label' },
+      },
+    },
+  },
+  args: {
+    id: 'unique-id',
+    label: {
+      content: 'Label',
+    },
+    hint: {
+      content: '',
```

Comment:
I have updated this with `children` since we dont have `content` with the react props like you suggested. Can you please check my PR, I have fixed your comment there and then maybe apply it here. Let me know what you think.

Reply:
I could make the changes now but I would wait for all of us to have the discussion in regards to this. I've created a [ticket ](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/21977)that tackles this. I think once we decide on a solution we can make this change across the board